fix for "error: size_t was not declared in this scope"
see also https://github.com/joachimwolff/sparse-neighbors-search/issues/10
author: Kenneth Hoste (HPC-UGent)
--- sparse-neighbors-search-0.7/sparse_neighbors_search/computation/typeDefinitionsBasic.h.orig	2023-02-13 18:33:53.877631417 +0100
+++ sparse-neighbors-search-0.7/sparse_neighbors_search/computation/typeDefinitionsBasic.h	2023-02-13 18:33:57.576622509 +0100
@@ -20,6 +20,7 @@
 #include <unordered_map>
 #include <utility>
 #include <limits>
+#include <stddef.h>
 
 #define MAX_VALUE std::numeric_limits<uint32_t>::max()
 
